Clarify Python prereq in Get started tutorial (#1869)
astronomer/docs: learn/get-started-with-airflow.md

## Prerequisites

- A terminal that accepts bash commands. This is pre-installed on most operating systems.
- [Docker Desktop](https://docs.docker.com/get-docker/) (v18.09 or higher).
- The [Astro CLI](https://docs.astronomer.io/astro/cli/install-cli).
- An integrated development environment (IDE) for Python development, such as [VSCode](https://code.visualstudio.com/).

> - Local installation of [Python 3](https://www.python.org/downloads/) is recommended to improve your local developer experience (e.g. with syntax highlighting), but not strictly necessary to use the Astro CLI.

Reviewer: ```suggestion
- Optional. A local installation of [Python 3](https://www.python.org/downloads/) to improve your Python developer experience.
```
